.travis.yml

```diff
@@ -99,12 +99,16 @@ script:
         cd "$BUILD_IN_DIR"
         cmake $CMAKE_ARGS "$SOURCE_DIRECTORY"
     fi
-  - make && sudo env PATH=$PATH make install && awesome --version && make check
+  - make && sudo env PATH=$PATH make install && awesome --version
   - |
-    if [ "$DO_COVERAGE" = "true" ]; then
-        make check-coverage || exit 1;
+    if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
```

Comment:
Ok, this variable really works the way you want it to:
https://travis-ci.org/psychon/awesome/builds/135217656
(Still I don't think it matters much if one continues after an error, but whatever...)

:+1: for this change